packages/console/src/App.tsx

```diff
@@ -51,6 +51,7 @@ import AppContent from './containers/AppContent';
 import ApiResourcePermissions from './pages/ApiResourceDetails/ApiResourcePermissions';
 import ApiResourceSettings from './pages/ApiResourceDetails/ApiResourceSettings';
 import CloudPreview from './pages/CloudPreview';
+import CloudPreviewAbout from './pages/CloudPreview/pages/About';
```

Comment:
Can you create an issue to track: change "cloud preview" to something else like "CloudWelcome" or "CloudGuide"

Reply:
Added, LOG-5418